hack/lib/common.sh

```diff
@@ -1,5 +1,43 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
Nit/Question: Is is typical to use `set -e` in a library? IIRC, anything that sources this script will inherit `set -e`.

Reply:
I removed the 'set -e' since that is causing an issues when sourced into other scripts that may have 'set -e pipefail' enabled.